src/components/found-user/found-user.ts

```diff
@@ -0,0 +1,20 @@
+import {Block} from '../../core/block';
+import {compileTemplateToElement} from '../../core/utils/compile-template';
+import {mapStateToPropsCallBack} from '../../store/utils';
+import templatePug from './found-user.pug';
+import './found-user.scss';
```

Comment:
```suggestion
import './found-user.scss';
import {Block} from '../../core/block';
import {compileTemplateToElement} from '../../core/utils/compile-template';
import {mapStateToPropsCallBack} from '../../store/utils';
import templatePug from './found-user.pug';
```

Обычно импорт стилей выносят, на самый верх